pkg/generator/caches_test.go

```diff
@@ -187,7 +187,8 @@ func createTestDataForIngress(
 		},
 	}, &translatedIngress)
 
-	caches.addStatusVirtualHost()
+	statusVirtualHost := statusVHost()
+	caches.statusVirtualHost = &statusVirtualHost
```

Comment:
Is this needed? Should be already done in this call, right?
https://github.com/knative-sandbox/net-kourier/pull/307/files#diff-99be96ee4d2233307511fe08afab35fce19f6bb870f9a92d140cee81a2fa3b4aR44

Reply:
Good call, fixed!